src/main/java/com/manardenza/console/ContentsUserMenu.java

```diff
@@ -0,0 +1,261 @@
+            if (arrival.after(departure) || arrival.before(new Date())) {
+                System.out.println("Arrival date must exceed current date but must come before departure!");
+                continue;
+            }
+            break;
+        }
+        Map<String, List<Room>> foundRooms = hotelController.getAvailableRooms(city, persons, price, arrival,
+            departure);
+        System.out.println("The list of rooms found:\n ");
+        VisualUserMenu.printMapInConsole(foundRooms);
+        VisualUserMenu.outputSplitLine();
+        bookRoomMenu(arrival, departure, foundRooms);
+        return foundRooms;
+    }
+
+    private void bookRoomMenu(Date reservedFrom, Date reservedTo, Map<String, List<Room>> foundRooms) {
+        List<String> foundHotelNames = new ArrayList<>();
+        foundRooms.forEach((key, value) -> foundHotelNames.add(key));
+        long reserveId = 0L;
+        System.out.println("Do you want to make a reservation?");
+        VisualUserMenu.printListInConsole(ListMenu.getBookingHeader(), ListMenu.getBookingMenu());
+        Integer action;
+        do {
+            action = Integer.parseInt(visualUserMenu.getValidInputFromUser("Choose action: ", InputType
+                .INTEGER));
+            switch (action) {
+                case (1):
+                    Integer hotelIndex;
+                    Integer roomIndex;
+                    do {
+                        hotelIndex = Integer.parseInt(visualUserMenu.getValidInputFromUser("Enter the number of "
+                            + "hotel: ", InputType.INTEGER));
+                    } while (!visualUserMenu.validateIntegerSize(foundHotelNames.size(), hotelIndex));
+                    String selectedHotelName = foundHotelNames.get(hotelIndex - 1);
+                    do {
+                        roomIndex = Integer.parseInt(visualUserMenu.getValidInputFromUser("Enter the number of "
+                           + "room: ", InputType.INTEGER));
+                    } while (!visualUserMenu.validateIntegerSize(foundRooms.get(selectedHotelName).size(), roomIndex));
+                    Hotel selectedHotel = hotelController.findHotelByName(selectedHotelName).get(0);
+                    Room selectedRoom = foundRooms.get(selectedHotelName).get(roomIndex - 1);
+                    reserveId = reservationController.bookRoom(reservedFrom,
+                        reservedTo, selectedRoom, selectedHotel);
+
+                    System.out.println("Room reserved successfully!\n You reservation ID: " + reserveId);
+                    break;
+                default:
+                    return;
+            }
+        } while (!visualUserMenu.validateIntegerSize(ListMenu.getBookingMenu().size(), action));
+    }
+
+    private void getAllUserReservationsMenu() {
+        List<Reservation> reservations = reservationController.getAllUserReservations();
+        if (reservations.isEmpty()) {
+            System.out.println("No reservations found");
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Replace this usage of System.out or System.err by a logger. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS106)